app/serializers/book_serializer.rb

```diff
@@ -1,4 +1,8 @@
 class BookSerializer < Panko::Serializer
-  attributes :author, :title, :genre, :editor, :year, :image, :slug
+  attributes :author, :title, :genre, :editor, :year, :image, :times_rented, :slug
+
+  def times_rented
+    object.rents.count.to_s
```

Comment:
Why string?

Reply:
I was testing the operation of object, and I forgot to remove "# {}", so when I run rubocop I leave it like this ... to_s.